gufe/tokenization.py

```diff
@@ -115,14 +115,17 @@ class GufeTokenizable(abc.ABC, metaclass=_ABCGufeClassMeta):
     This extra work in serializing is important for hashes that are stable
     *across different Python sessions*.
     """
+    def __repr__(self):
+        return f"<{self.key}>"
+
     def __lt__(self, other):
         return self.key < other.key
 
     def __eq__(self, other):
         if not isinstance(other, self.__class__):
             return False
 
-        return self.to_keyed_dict() == other.to_keyed_dict()
+        return self.key == other.key
```

Comment:
Should be -- I'm guessing that the history here was that this predated gufe keys and was a check on `to_dict()`, and then someone realized that `to_dict()` was really slow because it had to recurse through all objects, and sped it up with `to_keyed_dict()` after we had keys.

Just checking the key like this should be correct and faster, I think. Any dissent, @dotsdl?